sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultBackupClient.cs

```diff
@@ -68,12 +68,12 @@ public KeyVaultBackupClient(Uri vaultUri, TokenCredential credential, KeyVaultAd
         /// Initiates a full key backup of the Key Vault.
         /// </summary>
         /// <param name="blobStorageUri">The <see cref="Uri"/> for the blob storage resource.</param>
-        /// <param name="sasToken">A Shared Access Signature (SAS) token to authorize access to the blob.</param>
+        /// <param name="sasToken">Optional Shared Access Signature (SAS) token to authorize access to the blob. If null, Managed Identity will be used to authenticate instead.</param>
```

Comment:
How does the client automatically fallback to managed identity without constructing the client with the appropriate credential type?

Reply:
My understanding is that it was the service's managed identity, but I'll find out.

Reply:
It seems like this is only to say "use the user-assigned managed identity the Managed HSM was assigned to auth against storage", but we're trying to get confirmation. If confirmed, this would be enough as written.

Reply:
So long as there is no way this config can be missing, I agree.

Reply:
Talking with the service team, it just tells it to use the assigned user-managed identity. Any credential for the user/app is still used to communicate with KV/MHSM, so this is fine.

That said, there are some other issues with MHSM rollout, so I'll not be merging this just yet.